shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -275,10 +275,18 @@ - (void)removeSplashScreenViewIfPresent {
       }];
 }
 
-- (void)installSplashScreenViewCallback {
+- (void)onFlutterViewCallback {
```

Comment:
This works fine for a splash screen but isn't ideal for a next frame callback.  We should look at using PlatformView::SetNextFrameCallback

Reply:
It may also be worth adding something to NSNotificationCenter instead of rolling our own callback logic for this.

Reply:
This works not fine when there is not splash screen.And I fix it so that callback can be trigger whether splash screen exist or not.

Reply:
This is based on whether the viewcontroller has come into view, which is not the same as whether the engine has actually rendered a frame

Reply:
For viewcontroller, we need this callback to determine whether flutter view has rendered, but now this is not available if there is not splash screen.